content/docs/aro/blob-storage-csi/index.md

```diff
@@ -0,0 +1,315 @@
+    oc new-project ${CSI_TESTING_PROJECT}
+    ```
+
+
+1. Now, you are ready to create the storage class, the persistent volume claim and the testing pod. 
+
+    ```bash
+    cat <<EOF | oc apply -f -
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: blob
+    provisioner: blob.csi.azure.com
+    parameters:
+      resourceGroup:  $RG_NAME
+      storageAccount: $STORAGE_ACCOUNT_NAME
+      containerName:  $BLOB_CONTAINER_NAME
+      # server: SERVER_ADDRESS  # optional, provide a new address to replace default "accountname.blob.core.windows.net"
+    reclaimPolicy: Retain  # if set as "Delete" container would be removed after pvc deletion
+    volumeBindingMode: Immediate
+    mountOptions:
+      - -o allow_other
+      - --file-cache-timeout-in-seconds=120
+      - --use-attr-cache=true
+      - -o attr_timeout=120
+      - -o entry_timeout=120
+      - -o negative_timeout=120
+    EOF
+
+    cat <<EOF | oc apply -f -
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: pvc-blob
+    spec:
+      accessModes:
+        - ReadWriteMany
+      resources:
+        requests:
+          storage: 10Gi
+      storageClassName: blob
+    EOF
+
+    cat <<EOF | oc apply -f -
+    kind: Pod
+    apiVersion: v1
+    metadata:
+      name: nginx-blob
+    spec:
+      nodeSelector:
+        "kubernetes.io/os": linux
+      containers:
+        - image: mcr.microsoft.com/oss/nginx/nginx:1.17.3-alpine
+          name: nginx-blob
+          command:
+            - "/bin/sh"
+            - "-c"
+            - while true; do echo $(date) >> /mnt/blob/outfile; sleep 1; done
+          volumeMounts:
+            - name: blob01
+              mountPath: "/mnt/blob"
+      volumes:
+        - name: blob01
+          persistentVolumeClaim:
+            claimName: pvc-blob
+    EOF
+    ```
+
+1. Wait until the Pod is created
+    
+    ```bash
+    oc get po
+    ```
+
+1. Get a shell inside the pod and view the file in the blob storage
+    
+    ```bash
+    oc exec -it nginx-blob -- sh
+    df -h
+    ls -al /mnt/blob/outfile
+    cat /mnt/blob/outfile
+    ```
+1. You should see an output like this:
+
+    ![Image](images/blob-test.png)
+
+
+## Clean up
+
+This section is to delete all the resources created with this guideline. 
+
+   ```bash
+   oc delete pod nginx-blob -n ${CSI_TESTING_PROJECT}
+   oc delete project ${CSI_TESTING_PROJECT}
+   helm uninstall blob-csi-driver -n ${CSI_BLOB_PROJECT}
+   oc delete project ${CSI_BLOB_PROJECT}
+   oc delete pvc pvc-blob
+   oc delete sc blob
+   oc delete pv $(oc get pv -o json | jq -r '.items[] | select(.spec.csi.driver | test("blob.csi.azure.com")).metadata.name')
+   az storage account delete --name $STORAGE_ACCOUNT_NAME -g $RG_NAME 
+   az ad app delete --id $(az ad app list --app-id=${AAD_CLIENT_ID} | jq -r '.[0].id')
+
+   ```
+
+
+References:
+
+- https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/charts/README.md
```

Comment:
I would note that we're using this as a source right at the top of the document vs at the end.

Reply:
moved to the top of the document.